src/main/java/seedu/address/model/question/Question.java

```diff
@@ -32,6 +31,8 @@ public Question(Name name, Importance importance, Set<Tag> tags, Set<Choice> cho
         this.importance = importance;
         this.tags.addAll(tags);
         this.choices.addAll(choices);
+        this.randomisedChoices.addAll(choices);
+        Collections.shuffle(randomisedChoices);
```

Comment:
Rather than randomising here, would it be better to set and shuffle the choices in getRandomisedChoices instead? If we shuffle the choices here each question will always return the same set of 'randomised' choices, which seems to defeat the purpose of randomising